components/Slider/tests/unit/SliderTests.m

```diff
@@ -913,6 +913,42 @@ - (void)testThumbElevationDefault {
   XCTAssertEqual(self.slider.thumbElevation, MDCShadowElevationNone);
 }
 
+- (void)testThumbRadiusForState {
```

Comment:
Please test the default/fallback behavior as well.

Reply:
Done

Reply:
I don't see the test for the fallback behavior.  Could you please add a test similar to those in cl/194783040 where each UIControlState is checked?

Reply:
Done with `-testThumbRadiusDefaultWhenStatefulAPIEnabled`